package.json

```diff
@@ -10,8 +10,8 @@
     "test:coverage": "jest --coverage",
     "precommit": "lint-staged && npm test",
     "prepush": "npm test",
-    "start": "webpack --mode production src/",
-    "start:dev": "webpack-dev-server --mode development src/ --content-base public/"
+    "start": "webpack --config webpack.prod.js && node server.js",
```

Comment:
webpack is a build tool. It is not supposed to be run on production. You can create a bundle.js and use that file in your index.html